### ISSUE TITLE
tye run --docker fails on Windows Containers

### ISSUE THREAD

**Issue by Jojojov** [dotnet/tye#626]
### Describe the bug

When I start an application (frontend/backend) on  a Windows 1 machine with Windows containers using 'tye run --docker', an error follows:

Creating docker network tye_network_09aad465-4
Running docker command network create --driver bridge tye_network_09aad465-4
Running docker command with exception info  Error response from daemon: could not find plugin bridge in v1 plugin registry: plugin not found

I believe in that situation 'docker network create --driver nat tye_network_09aad465-4" should be used ('nat' instead of 'bridge')?

### To Reproduce
<!--
What steps can we follow to reproduce the issue?

run a service on a Windos 10 machine with Windows containers using 'tye run --docker'

### Further technical details

- Include the output of `tye --version`

       0.4.0-alpha.20371.1+d7623120d66b85bfeda8ab69eb5ff792df6b7243

- The platform (Linux/macOS/Windows)

       Windows 10 / Docker Desktop version 2.3.0.4 (46911)

**Comment by spboyer** [comment]
Is there a business reason you are using Windows containers?

**Comment by Jojojov** [comment]
Not really, just experimenting with containerising apps that use windows-specific libraries.

**Comment by recumbent** [comment]
One reason to want to user a windows container is that that is the only container option for the CosmosDB emulator (so far as I'm aware). Even more so since an installed instance of the emulator requires elevation to run which causes Tye to fail...

**Comment by spboyer** [comment]
@recumbent - [Linux work is currently underway for CosmosDB](https://feedback.azure.com/forums/263030/suggestions/18533509). However, there is a container you can use for a CosmosDB emulator on Linux. -https://hub.docker.com/r/ridhoq/cosmosdb-server

This is based on the [vercel/cosmos-db](https://github.com/vercel/cosmosdb-server) local emulator written in Tyepscript. Using this image would allow you to run all your containers as Linux based images.
